pkg/update_pod_status_util.go

```diff
@@ -528,6 +471,60 @@ func (initp *InitParams) wipeVolumes(dirtyVolumes, nsDevicePaths, nsFilePaths []
 	return dirtyVolumes, nil
 }
 
+func (initp *InitParams) cleanVolume(volume *Volume, wg *sync.WaitGroup, guard chan struct{},
+	isInitializing bool) error {
+	effectiveMethod := volume.effectiveWipeMethod
+
+	if isInitializing {
+		effectiveMethod = volume.effectiveInitMethod
+	}
+
```

Comment:
Add a logging statement here saying --> Cleaning up block volume, volumeName, cleanupMethod.

This func is called from three places. Caller can have a top level log message telling that its `initilize/wipe/cleanup dirty volume` then this func can log volume name and cleanup method